tests/integration/rules/unique.phpt

```diff
@@ -0,0 +1,39 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Exceptions\NestedValidationException;
+use Respect\Validation\Exceptions\UniqueException;
+use Respect\Validation\Validator as v;
+
+try {
+    v::unique()->check([1, 2, 2, 3]);
+} catch (UniqueException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    V::not(v::unique())->check([1, 2, 3, 4]);
+} catch (UniqueException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::unique()->assert('test');
+} catch (NestedValidationException $exception) {
+    echo $exception->getFullMessage().PHP_EOL;
+}
+
+try {
+    v::not(v::unique())->assert(['a', 'b', 'c']);
+} catch (NestedValidationException $exception) {
+    echo $exception->getFullMessage();
```

Comment:
I don't know if this is a big problem, but this don't have the `PHP_EOL` statement.